nifty-renderer-libgdx/src/main/java/de/lessvoid/nifty/gdx/render/GdxGL.java

```diff
@@ -1,10 +1,11 @@
 package de.lessvoid.nifty.gdx.render;
 
 import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.graphics.GL10;
-import com.badlogic.gdx.graphics.GL11;
 
+import com.badlogic.gdx.utils.BufferUtils;
 import de.lessvoid.nifty.render.batch.spi.GL;
+import org.lwjgl.opengl.GL11;
```

Comment:
Problem is, that methods like "glAlphaFunc" are not supported in GL20 and Gdx.gl is purely GL20.
I will change as much methods as possible, but do you have a suggestion what to do with unsupported ones?